### PR TITLE
Gitops 1

### PR DESCRIPTION
**Pull request by pmithil7**

### CODE REVIEW

File: argocd/pmithil7-project.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: <USERNAME>
```
Comment (jessesuen) [inline argocd/pmithil7-project.yaml:4]: This should be your github handle

File: argocd/pmithil7-project.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+spec:
+  sourceRepos:
+    - '*'
+  destinations:
+    - namespace: 'service-pmithil7-*'
+      server: http://cluster-argocon:8001
+  roles:
+  - name: admin
+    policies:
+    - p, proj:<USERNAME>:admin, *, *, <USERNAME>/*, allow
```
Comment (nikita-akuity) [inline argocd/pmithil7-project.yaml:14]: Please replace <USERNAME> with  your github name

**Review by nikita-akuity — CHANGES_REQUESTED** [review]

File: argocd/pmithil7-project.yaml (hunk 2 above)
Comment (jessesuen) [inline argocd/pmithil7-project.yaml:14]: username should be replaced with githubhandle

File: argocd/pmithil7-project.yaml (hunk 1 above)
Comment (pmithil7) [reply argocd/pmithil7-project.yaml]: done

File: argocd/pmithil7-project.yaml (hunk 2 above)
Comment (pmithil7) [reply argocd/pmithil7-project.yaml]: done

**Review by jessesuen — APPROVED** [review]

**Pull request merged**